modules/developer_manual/pages/_partials/app/fundamentals/complete-info.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0"?>
+<info>
+    <!-- Mandatory fields -->
+    <id>yourappname</id>
+    <name>Your App</name>
+    <description>Your application description</description>
+    <version>1.0</version>
+    <licence>AGPL</licence>
+    <screenshot small-thumbnail="https://raw.githubusercontent.com/foo/myapp/master/screenshots/thumb.png"
+        >https://raw.githubusercontent.com/foo/myapp/master/screenshots/big.png</screenshot>
+    <!-- Category values available at: https://marketplace.owncloud.com/ajax/categories -->
+    <category>A category for the application. </category>
+    <summary>A summary of the application's purpose (max 90 chars)</summary>
+
+    <types>
+        <filesystem/>
+    </types>
+
+    <documentation>
+        <user>https://doc.owncloud.com/server/latest/user_manual/pim/contacts.html</user>
+        <admin>https://doc.owncloud.com/server/latest/admin_manual/configuration_server/occ_command.html?highlight=contact#dav-commands</admin>
+        <developer>https://github.com/owncloud/contacts/blob/master/README.md</developer>
+    </documentation>
+
+    <author>Your Name</author>
+    <namespace>YourapplicationsNamespace</namespace>
+    <website>https://owncloud.org</website>
+    <bugs>https://github.com/owncloud/theapp/issues</bugs>
+    <repository type="git">https://github.com/owncloud/theapplication.git</repository>
+    <ocsid>1234</ocsid>
+
+    <dependencies>
+        <php min-version="5.4" max-version="5.5"/>
+        <database>sqlite</database>
+        <database>mysql</database>
+        <command os="linux">grep</command>
+        <command os="windows">notepad.exe</command>
+        <lib min-version="1.2">xml</lib>
+        <lib max-version="2.0">intl</lib>
+        <lib>curl</lib>
+        <os>Linux</os>
+        <owncloud min-version="6.0.4" max-version="8"/>
+    </dependencies>
+
+	<repair-steps>
+		<install>
+			<step>OCA\FilesEncryption\Migration\Install</step>
```

Comment:
this sample should be consistent - name space is 'YourapplicationsNamespace' - so this should be OCA\YourapplicationsNamespace\...